src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -263,6 +373,18 @@ private static FormulaValue ReplaceBlankWithEmptyString(IRContext irContext, int
             }
         }
 
+        private static FormulaValue ExactValueTypeOrTable<T>(IRContext irContext, int index, FormulaValue arg) where T : FormulaValue
```

Comment:
`ExactValueTyoeOrTableOrBlank`? Blanks are also allowed in this method

Reply:
Done